docs/source/triggers/index.rst

```diff
@@ -134,21 +134,33 @@ as well as retrieve the commands in flight with ``getCommandsInFlight``.
 Like ``Scenario`` or ``Update``, you can use ``do`` notation and
 ``getTime`` with ``TriggerA``.
 
-We can specify the templates that our trigger will operate
+We can specify the templates and interfaces that our trigger will operate
 on. In our case, we will simply specify ``AllInDar`` which means that
-the trigger will receive events for all template types defined in the
-DAR. It is also possible to specify an explicit list of templates. For example,
-to specify just the ``Message`` template, one would write:
+the trigger will receive events for all template and interface types defined in the
+DAR.
+
+It is also possible to specify an explicit list of templates and interfaces. For example,
+to only receive events for the ``Message`` template, one would write:
 
 .. code-block:: daml
 
    ...
    registeredTemplates = RegisteredTemplates [registeredTemplate @Message],
    ...
 
-This is mainly useful for performance reasons if your DAR contains many templates that are not relevant for your trigger.
-Note that providing an explicit list of templates also filters the result of querying the ACS using the Trigger API:
-contracts of the excluded templates cannot be queried.
+and to specify events for both the ``Message`` template and (say) an interface called ``Identity`` (which
+here we will imagine as being implemented by the template ``Alias``), one would write:
+
+.. code-block:: daml
+
+   ...
+   registeredTemplates = RegisteredTemplates [registeredTemplate @Message, registeredTemplate @Identity],
+   ...
+
+This is mainly useful for performance reasons if your DAR contains many templates and interfaces that are not relevant
+for your trigger.
+Note that providing an explicit list of templates and interfaces also filters the result of querying the ACS using
+the Trigger API: contracts of the excluded templates and interfaces cannot be queried.
```

Comment:
I believe the rest of the documentation is grounded on a specific example. Here instead you are asking the reader to imagine something. I believe this might cause some confusion. If that's the case, I would probably either:
- make interfaces part of the getting started and make changes here to reflect this
   - this is more work and I'm not 100% sure it makes too much sense in the context of the GSG
- describe the possibility of subscribing to multiple interfaces and templates (or a mix of both) as a separate note
   - this is probably simpler and still carries the point home IMO
```suggestion

This is mainly useful for performance reasons if your DAR contains many templates and interfaces that are not relevant
for your trigger.
Note that providing an explicit list of templates and interfaces also filters the result of querying the ACS using
the Trigger API: contracts of the excluded templates and interfaces cannot be queried.

.. note::

    In these examples we used templates. Note that interfaces can be passed as well wherever a template is passed, using the same `RegisteredTemplates` type. You are free to pass multiple templates and interfaces and possibly mix the two freely in a single request.
```

Reply:
Thanks for that suggestion @stefanobaghino-da. Totally agree wrt not expanding the current example and trying to be lighter handed.